docs/logging-and-managing-experiment-results/updating-existing-experiment.rst

```diff
@@ -0,0 +1,139 @@
+.. _update-existing-experiment:
+
+Updating existing experiment
+============================
+[loom-placeholder]
+
+You can update experiments even after they finished running. This lets you add new data or visualizations to the previously closed experiment and makes multi-stage training convenient.
+
+.. _update-existing-experiment-basics:
+
+Why you may want to update an existing experiment?
+--------------------------------------------------
+Updating existing experiment can come in handy in several situations:
+
+* You want to add metrics or visualizations to the closed experiment.
+* You finished model training and closed the experiment earlier, but now you want to continue training from that moment. Actually, you can even make multiple iterations of the procedure: ``resume experiment -> log more data``. Have a look at the :ref:`simple example <update-existing-experiment-basics-simple-example>` below for details.
+
+.. _update-existing-experiment-basics-simple-example:
+
+How to update existing experiment?
+----------------------------------
+Check how to upload more data to the existing experiment that was previously closed.
```

Comment:
Instead of this Check out.. I think we coudl be more prescriptive. 
To update the experiment you need to fetch the experiment object from the project where this experiment is. 

To do that you need to get the project

get the experiment


Something like that.

Reply:
... or just show it in one snippet as you did just say that you need to get project and experiment from that project

Reply:
ok, corrected.